content/articles/working-with-styled-components-in-react/index.md

```diff
@@ -0,0 +1,464 @@
+---
+layout: engineering-education
+status: publish
+published:
+url: /working-with-styled-components-in-react/
+title: Working with Styled-components in React
+description: Styling a simple web page in React using Styled-components.
+author: elly-omondi
+date: 2021-06-17T00:00:00-9:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/working-with-styled-components-in-react/hero.jpg
+    alt: Styled-components
+---
+
+### Prerequisites
+Be sure to have the following requirements installed and running to follow the coding part.
+
+1. Command Line Interface running on administrative privileges.
+
+2. Basic knowledge of [JavaScript programming language](https://www.w3schools.com/js/DEFAULT.asp) and [CSS](https://www.w3schools.com/css/).
+
+3. A package manager, [yarn](https://yarnpkg.com/) or [npm](https://www.npmjs.com/get-npm).
+
+4. Code Editor of your choice. I will be using [VS Code](https://code.visualstudio.com/download).
+
+5. [Node.js](https://www.nodejs.org). You can download the latest version.
+
+
+If you have worked with React in developing web applications, you will agree with me that adding style to your Components is super cool when using the styled-components feature. In any case, you have not used React or styled-components, do not worry because this article is for you.
+
+In this tutorial, you will learn an overview of React and what styled-components are, and why use it. In the end, we will build a simple web page in React and add styles to it using styled-components.
+
+Let's get it started!
+
+### Table of content
+This article will cover:
+- [What is React?](#What-is-react)
+- [What is Styled-components](#What-is-Styled-components)
+- [Advantages of using Styled-components](#Advantages-of-using-styled-components)
+- [Creating and styling a simple web-page using styled-components](#Styling-a-simple-webpage)
+
+
+
+### What is React
+React is a JavaScript library built and maintained by Facebook. React is purely Open-source.
```

Comment:
Change the second sentence, purely open-source. Insert it in the first sentence as: React is an open-source JavaScript library built and maintained by Facebook.